evergreen/config.yml

```diff
@@ -384,6 +384,14 @@ tasks:
         export CURL=${curl|curl}
 
         if [[ -n "${run_tests_against_baas|}" ]]; then
+            if [[ -z "${baas_secrets_aws_access_key|}" || -z "${baas_secrets_aws_secret_key|}" ]]; then
```

Comment:
Where do these come from? Are they custom project variables that have been added realm-core in evergreen or are they common to all projects? If they are common, is there a variable reference sheet somewhere or how did you find them?

Reply:
Ok thanks!